### ISSUE TITLE
Add laplacian gradient normalisation

### ISSUE THREAD

**Issue by justinpinkney** [justinpinkney/sumie#4]
As an alternative to fft parameterisation of the image as this causes various issues when scaling/interpolating.

**Comment by justinpinkney** [comment]
A very effective and simpler alternative is to use a pyramid parameterisation of the image rather than modifying the gradients. See https://github.com/tensorflow/lucid/blob/bccdc6415801349099e19a5003ff9ceb506856f7/lucid/optvis/param/lowres.py
